tests/modules/nf-core/hifiasm/main.nf

```diff
@@ -41,6 +41,34 @@ workflow test_hifiasm_with_hic {
     hic_reads1 = file(params.test_data['homo_sapiens']['illumina']['test_1_fastq_gz'], checkIfExists: true)
     hic_reads2 = file(params.test_data['homo_sapiens']['illumina']['test_2_fastq_gz'], checkIfExists: true)
 
-    HIFIASM ( input, [], [], hic_reads1, hic_reads2 )
+    HIFIASM ( input, [], [], hic_reads1, hic_reads2, [], [], [] )
+}
+
+
+/*
+ * Test with Nanopore ultra-long reads
+ */
+workflow test_hifiasm_with_ul {
+    input = [
+        [ id:'test' ], // meta map
+        [ file(params.test_data['homo_sapiens']['pacbio']['hifi'], checkIfExists: true) ]
+    ]
+    ont_ul = file(params.test_data['homo_sapiens']['illumina']['test_2_fastq_gz'], checkIfExists: true)
```

Comment:
Why are you using an illumina data set for ont ultra long input?

Reply:
Because there is no ul for human available in the test dataset and this works. Not ideal, but do the work